.github/workflows/linux.yml

```
@@ -5,16 +5,38 @@ on:
   push:
     branches: [ main ]
     tags: [ '*' ]
+  schedule:
+    - cron: '15 7 * * 0' # Sundays at 0715Z
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
+
   linux:
     name: Linux (latest)
     uses: ./.github/workflows/ci.yml
     with:
       id: linux-latest
       runner: ubuntu-latest
       container: archlinux/archlinux:latest
+
+  issue_bot:
+    name: Issue bot
+    # if: ${{ github.event_name == 'schedule' && ( cancelled() || failure() ) }}
```

uncomment after verifying issues are opened
```suggestion
    if: ${{ github.event_name == 'schedule' && ( cancelled() || failure() ) }}
```